contracts/services/neutron-ibc-transfer/src/tests.rs

```diff
@@ -150,6 +161,86 @@ impl ServiceTestSuite<Empty, NeutronQuery> for IbcTransferTestSuite {
     }
 }
 
+#[allow(dead_code)]
+pub trait Neutron: Module<ExecT = Empty, QueryT = NeutronQuery, SudoT = SudoMsg> {}
+
+pub struct NeutronKeeper {}
+
+impl Neutron for NeutronKeeper {}
+
+impl NeutronKeeper {
+    pub fn new() -> Self {
+        Self {}
+    }
+}
+
+impl Module for NeutronKeeper {
+    type ExecT = Empty;
+    type QueryT = NeutronQuery;
+    type SudoT = SudoMsg;
```

Comment:
Maybe it's worth moving this to somewhere else so that it can be used for any Neutron specific contracts instead of having to copy it

Reply:
I suggest to do this later, when the need arises .. I don't want to delay the merge of this PR.